src/lib.rs

```diff
@@ -71,6 +71,13 @@ impl Default for Client {
             connector
         });
 
+        #[cfg(feature = "rustls")]
+        let connector = hyper_rustls::HttpsConnectorBuilder::new()
+            .with_native_roots()
+            .https_or_http()
+            .enable_http2()
+            .build();
```

Comment:
It seems that we should use `wrap_connector` here to provide the configured `connector` above.

Reply:
We'll also need to disable the enforcement of http on the connector here (that only happens in the `native-tls` connector as of now). I've opened a PR with all these changes here: https://github.com/loyd/clickhouse.rs/pull/102.